src/experiments/preferences/api.js

```diff
@@ -56,13 +37,26 @@ var preferences = class preferences extends ExtensionAPI {
           async getIntPref(name, defaultValue) {
             return Services.prefs.getIntPref(name, defaultValue);
           },
+          async setIntPref(name, defaultValue) {
+            return Services.prefs.setIntPref(name, defaultValue);
+          },
           async getBoolPref(name, defaultValue) {
             return Services.prefs.getBoolPref(name, defaultValue);
           },
+          async setBoolPref(name, defaultValue) {
+            return Services.prefs.setBoolPref(name, defaultValue);
+          },
+          async setCharPref(name, defaultValue) {
+            return Services.prefs.setCharPref(name, defaultValue);
```

Comment:
Note that it might be possible to replace all of these with Preferences.set, see [Preferences.jsm](https://searchfox.org/mozilla-central/source/toolkit/modules/Preferences.jsm)

Reply:
@nhnt11 Would you like me to make that change in this PR or file an issue for the next release?

Reply:
No, probably best not to change too much for this release. An issue is fine :)

Reply:
#173